packages/components/src/internal/actions.ts

```diff
@@ -3092,3 +3092,34 @@ export function createQueryGridModelFilteredBySample(
         };
     });
 }
+
+interface IClientSideMetricCountResponse {
+    count: number;
+}
+
+export function incrementClientSideMetricCount(
+    applicationName: string,
```

Comment:
Since `applicationName` is optional, should it be the second parameter and marked as optional here?

Reply:
done

Reply:
After discussing further with Josh, the plan is to change the metric JSON format to layer the metrics within the application / feature area groups. With that, I'll move this param back to first, rename it to `featureArea` make it required.